nvm.sh

```diff
@@ -1044,6 +1044,20 @@ nvm_install_node_source() {
     make='gmake'
     MAKE_CXX="CXX=c++"
   fi
+
+  if [ "_$NVM_OS" = "_linux" ]; then
```

Comment:
I think "has binary" is less complex than "assuming because the OS is X that it has Y binary" - the `gmake` test should probably be checking that too.